nix/mobile/default.nix

```diff
@@ -9,19 +9,52 @@ let
     version = "10.2.1";
   };
   xcodeWrapper = composeXcodeWrapper xcodewrapperArgs;
-  androidPlatform = callPackage ./android.nix { inherit config gradle; };
-  selectedSources =
-    [ status-go ] ++
-    lib.optional platform.targetAndroid androidPlatform;
+  androidPlatform = callPackage ./android { inherit config pkgs target-os mkFilter nodejs maven localMavenRepoBuilder projectNodePackage prod-build-fn; status-go = status-go.android; };
+  iosPlatform = callPackage ./ios { inherit config pkgs mkFilter xcodeWrapper projectNodePackage; status-go = status-go.ios; };
+  fastlane = callPackage ./fastlane { inherit stdenv target-os; };
+  selectedSources = [
+      fastlane
+      status-go.android
+      status-go.ios
+    ] ++
+    lib.optional platform.targetAndroid androidPlatform ++
+    lib.optional platform.targetIOS iosPlatform;
+
+  yarn2nix = import (fetchFromGitHub {
+    owner = "moretea";
+    repo = "yarn2nix";
+    rev = "3cc020e384ce2a439813adb7a0cc772a034d90bb";
+    sha256 = "0h2kzdfiw43rbiiffpqq9lkhvdv8mgzz2w29pzrxgv8d39x67vr9";
+    name = "yarn2nix-source";
+  }) { inherit pkgs nodejs yarn; };
+  projectNodePackage = yarn2nix.mkYarnPackage {
```

Comment:
Same here, I get that this invokes `yarn2nix` on our node dependencies, but all the given parameters seem to be hardcoded aside from `projectNodePackage`. This should be it's own file with clear comments on what it does.

Reply:
Yeah, I'll move yarn2nix and this expression to its own file.